content/en/docs/setup/additional-setup/external-controlplane/index.md

```diff
@@ -398,6 +398,7 @@ and installing needed webhooks, configmaps, and secrets on the remote cluster so
     istio-leader                           0      2m9s
     istio-namespace-controller-election    0      2m11s
     istio-sidecar-injector                 2      2m1s
+    istio-validation-controller-election   0      2m6s
```

Comment:
@howardjohn I noticed that when I run locally the validatingwebhookconfiguration is NOT there, but it is when we run in CI. You also mentioned in the env WG call that only the mutatingwebhook is needed on the remote cluster. I'm wondering:
1. Why is it not there when I run locally?
2. Why isn't it needed? How does the validation happen if it's not there?

Do you understand what's going on?

Reply:
Its probably because of all the fake sed stuff we do. With a real root cert, we don't have it since we don't need to patch the validatingwebhookconfiguration, which is why we don't have the configmap.

Why we don't have the actual validatingwebhookconfiguration itself, I am not entirely sure. But its almost certainly due to the fact the doc and doc test are not the same

Reply:
Also its not needed in the remote-only cluster, since there is no config. In the config cluster you want it. If you install the CRDs you probably ought to install the webhook as well though.

Reply:
Oh, actually, I see this is the configmap, not the webhook, but I suppose my questions still apply? Any idea why the configmap is there in CI, but not when running locally?

Also, the second question still applies if I understood what you said in the env WG, that only the mutatingwebhook is needed in the remote cluster.

Reply:
Yes, the test is doing fake stuff the doc doesn't do. Because we have no real certificate, we need to patch the webhook with our self-signed cert, and that results in th configmap.

If you are a remote ONLY cluster (no config), you don't *need* CRDs or webhook, since they do nothing. If you are a remote-config cluster, you still want them

Reply:
That all makes sense. Thanks!